.github/workflows/ci.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
           python3 -m pip install tox-gh-actions
 
       - name: Run Linters
-        uses: ./.github/actions/pre-commit-action
+        uses: pre-commit/action@v3.0.0
```

Comment:
i maybe missed the discussion or the reason, why did we use the action inside our repo? Would it also make sense to adapt all other occurences of "./.github/actions/pre-commit-action" inside our repos?

Reply:
It is already partially done. I will take care about other one and will create task for it